### ISSUE TITLE
Feature request: Captions

### ISSUE THREAD

**Issue by luptilu** [kiln/template-line-chart-race#153]
A free user has just asked about adding captions to the line chart race (they were wondering how to add a captions tab): 

https://secure.helpscout.net/conversation/949851106/7591/

Maybe we could add a captions tab like in the bar chart race.

**Comment by theduncanclark** [comment]
Yes, good idea. The plan was always to add this to LCR once we'd rolled it out successfully in BCR.

**Comment by luptilu** [comment]
Requested by a free user

https://secure.helpscout.net/conversation/988747300/9481/
